post-service/src/main/resources/application-external.yml

```
@@ -1,22 +1,25 @@
 eureka:
   client:
     serviceUrl:
-      defaultZone: http://localhost:${EUREKA_PORT}/eureka/
+      defaultZone: ${EUREKA_URL}
+#      defaultZone: http://localhost:19090/eureka/
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)